packages/client/src/components/Login.js

```diff
@@ -47,7 +71,10 @@ const Login = () => {
             placeholder="Email"
             value={user.email}
             onChange={updateUser}
+            onBlur={handleBlur}
+            border={errors.email && "1px solid #EE5F5F"}
```

Comment:
This is our own input, we can do better by passing in something like `error={errors.email}` and handle the UI change on the styled component.

Reply:
Yes! Thank you, we discussed handling this more cleanly but were not sure how...turns out we just needed to spend a bit more time with the styled-comp. docs